go/type.go

```diff
@@ -310,6 +310,10 @@ func (self *Compiler) compile_dict(n *VimNode) string {
 	}
 }
 
+func (self *Compiler) compile_parenexpr(n *VimNode) string {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
receiver name should be a reflection of its identity; don't use generic names such as "this" or "self"